src/components/cards/AssetCard.tsx

```diff
@@ -0,0 +1,323 @@
+import {
```

Comment:
imo we should rename this component to `EthCard` or something like that, I meant to make it more generic so we could pass in an address and go from there but dont think we care about that atm